script.js

```diff
@@ -0,0 +1,15 @@
+var play;
+var counter = 0;
+function play() {
+  if (counter % 2 == 0) {
+    alert("X")
+  }
+  else if(counter % 2 == 1) {
+    alert("O");
+  }
+  counter ++
+}
+
+//This wouldn't work for some reason.//
+var text = document.getElementById("topLeft").textContent;
```

Comment:
So the reason this didn't work is because your script tag is in the `head` of the document. Thus, the script runs *before* the body is loaded. Thus, any JS that works with the DOM will not work (the body, and the elements inside don't 'exist' yet as far as the browser is concerned).

The fix is to put your `script` tag as the last line inside the body (right before the `</body>` tag. We'll see that there's another way to fix this with jQuery.